aggregator_api/src/routes.rs

```diff
@@ -33,6 +33,7 @@ pub(super) async fn get_config(
     State(config): State<Arc<Config>>,
 ) -> Json<AggregatorApiConfig> {
     Json(AggregatorApiConfig {
+        protocol: "DAP-05",
```

Comment:
Janus skipped ahead to DAP-07
```suggestion
        protocol: "DAP-07",
```